docs/concepts/client-handshake.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Client handshake
+description: The client handshake is a mechanism that is used to resolve a request’s authentication state from “unknown” to definitively signed in or signed out.
+type: conceptual
+---
+
+# The client handshake
```

Comment:
Let's add a callout that this flow applies only to website browser traffic and not native apps.